### PR TITLE
 add scripts for chaos and performance tests

### PR DESCRIPTION
**Pull request by minlei98**
## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please describe the System Under Test.
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: performance/scripts/create-hosted-clusters.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+#check if exit code is 0 and print message accordingly
+if [[ $EXIT_CODE -eq 0 ]]; then
+  echo $CONTAINER_NAME recovered
+else
+  echo $CONTAINER_NAME did not recovered
+fi
+
+# Search for string in logs and print matching lines
+podman logs $CONTAINER_NAME
+
```
Comment (mukrishn) [inline performance/scripts/create-hosted-clusters.py]: file extension is `.py`, should be `.sh`

**Review by mukrishn — COMMENTED** [review]
Can you also add a readme on how to run the script including the required environment variables.

File: chaos/scripts/container_interrupt.sh [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    podman stop $CONTAINER_NAME
+    podman rm $CONTAINER_NAME
+    echo "Container $CONTAINER_NAME has been deleted"
+else
+    echo "Container $CONTAINER_NAME does not exist"
+fi
+
+cp /root/kubeconfig /tmp/kubeconfig_con
+cp /opt/sti_ext/test_driver/harness/alerts/chaos/alerts /tmp/alerts_con
+podman run --name $CONTAINER_NAME --net=host --env-host=true -v /tmp/kubeconfig_con:/home/krkn/.kube/config:Z -v /tmp/alerts_con:/home/kraken/config/alerts:Z -d quay.io/krkn-chaos/krkn-hub:container-scenarios
```
Comment (mukrishn) [inline chaos/scripts/container_interrupt.sh]: @minlei98 can we also make all the container image as variable ?

Comment (minlei98) [reply chaos/scripts/container_interrupt.sh]: We could leave as is for now, then adjust the scripts later.

**Review by mukrishn — APPROVED** [review]

**Pull request merged**